mm2src/mm2_bitcoin/keys/src/segwitaddress.rs

```diff
@@ -94,6 +106,10 @@ impl SegwitAddress {
                 32 => Some(SegwitAddrType::P2wsh),
                 _ => None,
             },
+            1 => match self.program.len() {
+                32 => Some(SegwitAddrType::P2tr),
+                _ => None,
+            },
```

Comment:
```suggestion
            1 if self.program.len() == 32 => Some(SegwitAddrType::P2tr),
```

Can also be applied to above case as well.